schema/mysql/v8/visibility/schema.sql

```diff
@@ -44,6 +44,7 @@ CREATE TABLE executions_visibility (
   TemporalSchedulePaused        BOOLEAN       GENERATED ALWAYS AS (search_attributes->"$.TemporalSchedulePaused"),
   TemporalNamespaceDivision     VARCHAR(255)  GENERATED ALWAYS AS (search_attributes->>"$.TemporalNamespaceDivision"),
   BuildIds                      JSON          GENERATED ALWAYS AS (search_attributes->"$.BuildIds"),
+  PauseActivityTypes            JSON          GENERATED ALWAYS AS (search_attributes->"$.PauseActivityTypes"),
```

Comment:
`PauseActivityTypes` -> `PausedActivityTypes`

I see this typo in several places, please fix them.

Reply:
Renamed to PausedEntities everywhere